{"name": "test_nonexistent_user", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 404\n +  where 403 = <Response [403]>.status_code", "trace": "def test_nonexistent_user():\n        response = requests.get(f\"{BASE_URL}/users/nonexistentuser12345\")\n>       assert response.status_code == 404\nE       assert 403 == 404\nE        +  where 403 = <Response [403]>.status_code\n\ntest/test_github_api.py:33: AssertionError"}, "start": 1760964505735, "stop": 1760964506214, "uuid": "41189678-b049-4e4f-b14e-0ed5bec275a6", "historyId": "f6a2bc5a8c2406bb6eb5248ae0554088", "testCaseId": "f6a2bc5a8c2406bb6eb5248ae0554088", "fullName": "test.test_github_api#test_nonexistent_user", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_github_api"}, {"name": "host", "value": "EU314602016234.local"}, {"name": "thread", "value": "5806-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_github_api"}], "titlePath": ["test", "test_github_api.py"]}